CMakeLists.txt

```
@@ -1,29 +1,19 @@
-cmake_minimum_required(VERSION 3.15...3.18)
+cmake_minimum_required(VERSION 3.4...3.19)
```

```suggestion
cmake_minimum_required(VERSION 3.15...3.19)
```

Oops, COMMAND_ECHO requires 3.15.